Bamboo/backend/bamboo-backend/src/routes/auth/initialise/initialise.controller.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable prettier/prettier */
+import { Controller, Post, Body, UseInterceptors, Res } from "@nestjs/common";
+import { FormDataInterceptor } from "nestjs-form-data/dist/interceptors/FormData.interceptor";
+import jwt from 'jsonwebtoken';
+import { JwtPayload } from "src/types/auth.jwt.types";
+import { InitialiseService } from "./initialise.service";
+
+@Controller('')
+export class InitialiseController {
+    constructor (
+        private initialiseService: InitialiseService
+    ) {
+    }
+
+    @Post()
+    @UseInterceptors(FormDataInterceptor)
+    async initialiseAccountType(
+        @Body() body,
+        @Res() res
+    ){
+        const decoded: JwtPayload = jwt.decode(body.jwt) as JwtPayload;
+        try {
+            const { accountJwt } = await this.initialiseService.initialiseAccount(
```

Comment:
I'll combine the JWT logic to a middleware or interceptor so this code can be isolated